src/components/MissionProfile.jsx

```diff
@@ -0,0 +1,25 @@
+import React, { useEffect, useState } from 'react';
+import { useSelector } from 'react-redux';
+
+const MissionProfile = () => {
+  const { missions } = useSelector((state) => state.missions);
+  const [reserved, setReserved] = useState([]);
+
+  useEffect(() => {
+    const temp = missions.filter((mission) => mission.reserved);
+    setReserved(temp);
+  }, [missions]);
+
+  return (
+    <div className="profileMission">
+      <h2>My Mission</h2>
+      {reserved.map((mission) => (
+        <span key={mission.mission_id}>
+          {mission.mission_name}
+        </span>
+      ))}
```

Comment:
[OPTIONAL] Well done overall 💯  but the My Profile section could be enhanced by adding a placeholder message when the "My Missions" or "My Rockets" lists are empty (no missions joined or no rockets reserved). It would go like this 👇🏽 

```
      {reserved.length > 0 ? (
        reserved.map((mission) => (
          <span key={mission.mission_id}>
            {mission.mission_name}
          </span>
        ))
      ) : (
        <span>No reserved missions</span>
      )}
```